docs/build/getting-started/fcl-quickstart.md

```diff
@@ -5,157 +5,502 @@ sidebar_label: Simple Frontend
+First, let's create our app and navigate to it with the following terminal commands. From the root of where you keep your source code:
 
-```zsh
+```bash
 npx create-react-app fcl-app-quickstart
 cd fcl-app-quickstart
 ```
 
-Open the new project in a new window in your editor.
+This command uses sets up a new React project named `fcl-app-quickstart`. Then, we navigate into the project directory.
 
-It comes with a default layout, but let's remove it in `src/App.js` with something simple. Copy and paste this:
+Open the new project in your editor.
 
-```tsx
+The default layout includes some boilerplate code that we don't need. Let's simplify `src/App.js` to start with a clean slate. Replace the contents of `src/App.js` with:
+
+```jsx
 // src/App.js
 
 import './App.css';
 
 function App() {
-    return (
-        <div className="App">
-            <div>FCL App Quickstart</div>
-        </div>
-    );
+  return (
+    <div className="App">
+      <div>FCL App Quickstart</div>
+    </div>
+  );
 }
 
 export default App;
 ```
 
-Now let's run our app with the following `npm` command:
+This code defines a simple `App` component that displays the text "FCL App Quickstart".
 
-```zsh
+Now, let's run our app with the following `npm` command:
+
+```bash
 npm start
 ```
 
-You'll see a blank page with `FCL App Quickstart` at the top.
+This will start the development server and open your app in the browser. You should see a page displaying `FCL App Quickstart`.
 
 ## Setting Up FCL
 
-In order to use FCL, we need to install it. Shut the server down then run the following to download the library and set it as a dependency in our project:
+To interact with the Flow blockchain, we need to install the Flow Client Library (FCL). Stop the development server by pressing `Ctrl+C` in the terminal, and then run the following command to install FCL:
 
-```
+```bash
 npm install @onflow/fcl --save
 ```
 
-Next we'll want to add to our FCL configuration. There's a lot you can do here, but for this simple example, let's configure `accessNode.api` to talk to the Testnet Flow Access Node. An Access Node serves as the primary point of interaction for clients, such as wallets, dapps, and other services, to communicate with the Flow network. It provides a gateway for these clients to submit transactions, query data, and retrieve information without having to connect to the entire network or maintain a full copy of the blockchain.
+This command installs FCL and adds it to your project's dependencies.
 
-For our example, we are going to point at a free Access Node provided by Flow. Add the following config code to your `src/App.js`:
+Next, we'll configure FCL to connect to the Flow Testnet. An Access Node serves as the primary point of interaction for clients to communicate with the Flow network. It provides a gateway for submitting transactions, querying data, and retrieving information.
```

Comment:
If we have links we can use for [Flow Testnet] and [Access Node], I'd add them here.